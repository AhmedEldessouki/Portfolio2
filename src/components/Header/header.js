import { Link } from 'gatsby';
import PropTypes from 'prop-types';
import React from 'react';
import { css } from '@emotion/core';

import GlobalStyles from '../../styles/globalStyles';

const Header = ({ siteTitle }) => (
  <header
    css={css`
      background: black;
      margin-bottom: 1.45rem;
    `}>
    <div
      css={css`
        margin: 0 auto;
        maxwidth: 960px;
        padding: 1.45rem 1.0875rem;
      `}>
      <h1 style={{ margin: 0 }}>
        <Link
          to='/'
          style={{
            color: `white`,
            textDecoration: `none`,
          }}>
          {siteTitle}
        </Link>
      </h1>
    </div>
  </header>
);

Header.propTypes = {
  siteTitle: PropTypes.string,
};

Header.defaultProps = {
  siteTitle: ``,
};

export default Header;
