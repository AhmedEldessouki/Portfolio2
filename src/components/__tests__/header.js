/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Header';

describe('Header', () => {
  expect(true).toBe(true);
});
