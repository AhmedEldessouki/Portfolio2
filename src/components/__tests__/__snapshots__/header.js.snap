// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  css={
    Object {
      "map": undefined,
      "name": "xvi3ds",
      "next": undefined,
      "styles": "
      background: black;
      margin-bottom: 1.45rem;
    ",
      "toString": [Function],
    }
  }
>
  <div
    css={
      Object {
        "map": undefined,
        "name": "1qla906",
        "next": undefined,
        "styles": "
        margin: 0 auto;
        maxwidth: 960px;
        padding: 1.45rem 1.0875rem;
      ",
        "toString": [Function],
      }
    }
  >
    <h1
      style={
        Object {
          "margin": 0,
        }
      }
    >
      <a
        aria-current="page"
        className=""
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        Default Starter
      </a>
    </h1>
  </div>
</header>
`;
